src/app/containers/ConsentBanner/index.amp.jsx

```diff
@@ -64,12 +64,14 @@ const jsonInlinedScript = data => (
 
 // Style `amp-consent` as child due to inability to set
 // `layout` attribute on styled `amp-consent` component
+/* stylelint-disable selector-type-no-unknown */
 const AmpConsentWrapper = styled.div`
   & amp-consent {
```

Comment:
Is there any other way we can get this element than by its tag name?

If not, perhaps we could look at implementing something like this, to prevent stray disablers: https://stylelint.io/user-guide/rules/selector-type-no-unknown#optional-secondary-options

Reply:
All done :)